packages/core/src/machines/index.ts

```diff
@@ -635,27 +635,25 @@ export const createAuthMachine = ({
           }),
 
         autoSignIn: async () => {
-          if (typeof window !== 'undefined') {
-            const location = window.location
-            if (location.hash) {
-              const params = new URLSearchParams(location.hash.slice(1))
-              const refreshToken = params.get('refreshToken')
-              if (refreshToken) {
-                const session = await postRequest('/token', {
-                  refreshToken
-                })
-                // * remove hash from the current url after consumming the token
-                // TODO remove the hash. For the moment, it is kept to avoid regression from the current SDK.
-                // * Then, only `refreshToken` will be in the hash, while `type` will be sent by hasura-auth as a query parameter
-                // window.history.pushState({}, '', location.pathname)
-                const channel = new BroadcastChannel('nhost')
-                // TODO broadcat session instead of token
-                channel.postMessage(refreshToken)
-                return { session }
-              }
-            }
+          if (typeof window === 'undefined' || !window.location)
```

Comment:
I'd say we should wrap this in curly braces so we could visually separate our if statements a bit.

We are using the [curly](https://eslint.org/docs/rules/curly) ESLint rule against this in the console.

Reply:
I'm fine with it, although we will have to change it in lots of place as I have this (bad) habit :)